doc/installation.rst

```diff
@@ -2,21 +2,31 @@
 Installation
 ############
 
-If you intend to use EOS without any modifications, then you should follow the steps outlined in `Installing with a Package Manager`_
-or `Installing with the Python Package Index (PyPI)`_.
-If you intend to modify EOS for your personal purpose, then you must follow the steps outlined in `Installing from Source`_.
+We distribute EOS as packaged software for end-users and also as source code for developers.
 
 
-*********************************
-Installing with a Package Manager
-*********************************
+*****
+Linux
+*****
+
+Installation with `pip3`
+========================
+
+This is the recommended installation method using the package installer `pip3`:
 
+::
+
+  pip3 install --user eoshep
 
-Ubuntu Linux
-============
+You can now use EOS via the Python module or as system commands.
+
+
+Installation with `apt` (Ubuntu)
```

Comment:
Ubuntu/Debian and similar?

Reply:
I like 'siblings' for derivatives of Debian.